testbom/pom.xml

```diff
@@ -53,6 +55,45 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <!-- This is a transitive dep of the model-test and subsystem-test frameworks
+                     where we want to fix the version to something more recent -->
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>${version.com.google.guava}</version>
```

Comment:
Could you also add `<scope>test</scope>`?

Reply:
Added.